src/css/styles.scss

```
@@ -541,8 +541,8 @@
 
   .mobile-misc-tools-container {
     position: absolute;
-    top: 5rem;
-    right: 0;
+    top: 4rem;
```

```suggestion
    top: calc(5rem - #{$editor-container-padding});
```

not sure to do this statically in scss